docs/MVS_parameters_list.csv

```diff
@@ -1,54 +1,54 @@
+None,"Users can assign a project ID as per their preference.",1,"Cannot be the same as an already existing project","str",None,"project_id","projectid-label","project_data",
+None,"Users can assign a project name as per their preference.","Borg Havn",None,"str",None,"project_name","projectname-label","project_data",
+0,"The share of renewables in the generation mix of the energy supplied by the DSO (utility).",0.1,"Real number between 0 and 1","numeric","Factor","renewable_share","renshare-label","providers",
+"False","Allow the user to tag as asset as renewable.","True","Acceptable values are either True or False","boolean",None,"renewableAsset","renewableasset-label","production",
+None,"Brief description of the scenario being simulated.","This scenario simulates a sector-coupled energy system",None,"str",None,"scenario_description","scenariodescription-label","project_data",
+None,"Users can assign a scenario id as per their preference.",1,"Cannot be the same as an already existing scenario within the project","str",None,"scenario_id","scenarioid-label","project_data",
+None,"Users can assign a scenario name as per their preference.","Warehouse 14",None,"str",None,"scenario_name","scenarioname-label","project_data",
+None,"The level of charge (as a factor of the actual capacity) in the storage in the zeroth time-step.",":code:`storage capacity`: None, :code:`input power`: NaN, :code:`output power`: NaN","Acceptable values are either None or the factor. Only the column :code:`storage capacity` requires a value, in column :code:`input power` and :code:`output power` :code:`soc_initial` should be set to NaN. The :code:`soc_initial` has to be within the [0,1] interval.","numeric","None or factor","soc_initial","socin-label","storage_csv",
+0,"The maximum permissible level of charge in the battery (generally, it is when the battery is filled to its nominal capacity), represented by the value 1.0. Users can  also specify a certain value as a factor of the actual capacity.",":code:`storage capacity`: 1, :code:`input power`: NaN, :code:`output power`: NaN","Only the column :code:`storage capacity` requires a value, in column :code:`input power` and :code:`output power` :code:`soc_max` should be set to NaN. The :code:`soc_max` has to be in the [0,1] interval.","numeric","Factor","soc_max","socmax-label","storage_csv",
+1,"The minimum permissible level of charge in the battery as a factor of the nominal capacity of the battery.",":code:`storage capacity`:0.2, :code:`input power`: NaN, :code:`output power`: NaN","Only the column :code:`storage capacity` requires a value, in column :code:`input power` and :code:`output power` :code:`soc_min` should be set to NaN. The soc_min has to be in the [0,1] interval.","numeric","Factor","soc_min","socmin-label","storage_csv",
```

Comment:
```suggestion
0,"The minimum permissible level of charge in the battery as a factor of the nominal capacity of the battery.",":code:`storage capacity`:0.2, :code:`input power`: NaN, :code:`output power`: NaN","Only the column :code:`storage capacity` requires a value, in column :code:`input power` and :code:`output power` :code:`soc_min` should be set to NaN. The soc_min has to be in the [0,1] interval.","numeric","Factor","soc_min","socmin-label","storage_csv",
```

0 because it's the minimum SOC